service.js

```diff
@@ -1,67 +1,15 @@
 const bodyParser = require('body-parser');
 const express = require('express');
-const jsdom = require('jsdom');
-const urlparse = require('url');
-const {getMetadata} = require('page-metadata-parser');
-
-
-function buildObj(pairs) {
-  return pairs.reduce((newObj, [key, value]) => {
-    newObj[key] = value;
-    return newObj;
-  }, {});
-}
-
-
-function getDocumentMetadata(url, doc) {
-  let metadata = getMetadata(doc);
-
-  metadata.url = url;
-  metadata.original_url = url;
-  metadata.provider_url = url;
-
-  if (!metadata.favicon_url) {
-    const parsedUrl = urlparse.parse(url);
-    metadata.favicon_url = parsedUrl.protocol + '//' + parsedUrl.host + '/favicon.ico';
-  }
-
-  metadata.image = metadata.image && metadata.image.replace(/^\/\//, 'https://');
-
-  return metadata;
-}
-
-
-function getUrlMetadata(url) {
-  return new Promise((resolve, reject) => {
-    jsdom.env({
-      url: url,
-      done: function(err, window) {
-        if (!window) {
-          resolve({});
-          return;
-        }
-
-        resolve(getDocumentMetadata(url, window.document));
-      }
-    });
-  });
-}
```

Comment:
✂️ I cut all this stuff and moved it into [./lib/index.js](https://github.com/pdehaan/page-metadata-service/blob/db49331e01eef92114cd640e38fe672c8d629388/lib/index.js) so it could be shared with the CLI version.